packages/styles/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@trezor/styles",
+    "version": "1.0.0",
+    "private": true,
+    "license": "SEE ROOT LICENSE.md",
+    "main": "src/index",
+    "type": "module",
+    "sideEffects": false,
+    "scripts": {
+        "type-check": "tsc --build",
+        "lint": "eslint '**/*.{ts,tsx,js}'",
+        "test:unit": "jest"
+    },
+    "dependencies": {
+        "@mobily/ts-belt": "^3.10.0",
```

Comment:
This library looks really cool, but not that well adopted (exceeded tens of downloads per week just a few months back) 🤔

Reply:
Agree, but I will try to explain. I want to use some utility library, because I am sure it will eliminate/simplify lot of code   if used well. My two must have criteria is perfect TS support and FP support. 

In most of my previous project I used `ramda` which is very popular but TS support is quite shitty, we always had problems with that. Same goes for most alternatives like `rambda`, `remeda` and `lodash/fp` which usually have problems with TS and some of them could be quite slow. Another options was `fp-ts` which could do anything and has first class TS support, but it's quite "academic" and it's hard for people to get it.

This library that I picked is really cool. It is small, zero dependency, with perfect TS support, fastest that anything other etc. It is very easy to use and adopt for other people. But as you mentioned problem could be that library is not so popular, but I think it can work anyway because library is very simple in the end and we probably won't need to upgrade it for years and it should still work well, so there is very small risk of possible security breach.

In the end this library seems like only one that is really good compromise between standart utility library and some FP lib, which is also super-easy for even unexperience people to use. You can check this list https://project-awesome.org/stoeffel/awesome-fp-js where I checked nearly every library from the list 😅